src/EdFi.AnalyticsMiddleTier.DataStandard2/Base/MSSQL/0023-View-SectionDim-Create.sql

```diff
@@ -0,0 +1,46 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+
+IF EXISTS
+(
+    SELECT 1
+    FROM [INFORMATION_SCHEMA].[VIEWS]
+    WHERE [TABLE_SCHEMA] = 'analytics'
+          AND [TABLE_NAME] = 'SectionDim'
+)
+    BEGIN
+        DROP VIEW [analytics].[SectionDim];
+END;
+GO
+CREATE VIEW [analytics].[SectionDim]
+AS
+     SELECT DISTINCT 
+		  CAST([ssa].[SchoolId] AS VARCHAR) AS [SchoolKey],
+          CAST([ssa].[SchoolId] AS NVARCHAR) + '-' + [ssa].[ClassPeriodName] + '-' + [ssa].[ClassroomIdentificationCode] + '-' + [ssa].[LocalCourseCode] + '-' + CAST([ssa].[TermDescriptorId] AS NVARCHAR) + '-' + CAST([ssa].[SchoolYear] AS NVARCHAR) + '-' + [ssa].[UniqueSectionCode] + '-' + CAST([ssa].[SequenceOfCourse] AS NVARCHAR) AS [SectionKey],
+		  CONCAT([ssa].[LocalCourseCode],'-',ISNULL([Course].[CourseTitle], '')) AS [SectionName],
+		  ISNULL([Course].[CourseTitle], '') AS [SessionName],
+		  [ssa].[LocalCourseCode],
+		  ssa.SchoolYear, 
+		  s.ClassroomIdentificationCode as EducationalEnvironmentTypeId,
```

Comment:
Why are you using this alias? The section table has a column with the same name (EducationalEnvironmentTypeId)
In this case, add a join with EducationalEnvironmentType to show EducationalEnvironmentType.Description

Reply:
updated